apps/site/assets/ts/stop/components/StopPageHeaderRedesign.tsx

```diff
@@ -12,19 +12,26 @@ const StopPageHeaderRedesign = ({
 }): ReactElement<HTMLElement> => {
   return (
     <div className="u-bg--gray-bordered-background">
-      <header className="d-flex justify-content-space-between container">
+      <header
+        className="d-flex justify-content-space-between container"
+        style={{ alignItems: "center" }}
+      >
         <div className="d-flex">
           <h3>{stop && stop.name}</h3>
           <div className="ps-16 mt-12">
             {stop && routes && <StopFeatures stop={stop} routes={routes} />}
           </div>
         </div>
-        <div className="mt-20">
-          {stop && routes && routesHasBusRoute(routes) && `Stop ${stop.id}`}
+        <div>
+          {stop &&
+            routes &&
+            routesHasBusRoute(routes) &&
```

Comment:
This is an edge case but it _does_ happen: sometimes a stop has no service and no routes for it! But we would still want to show the stop ID. 
**suggestion:** remove the route check and solely rely on the `stop["station?"]` logic for rendering the stop ID.